src/main/webapp/style.css

```diff
@@ -1,8 +1,14 @@
 h1 {
-  font-family: Verdana, sans-serif; 
+  font-family: Verdana, sans-serif;
   font-size: x-large;
 }
```

Comment:
Consider creating a class for this styling to [avoid using type selectors](https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors) or consider applying the font to a more global element like `body` to keep the font consistent between different text elements.

Reply:
Yeah def, I think this is here from rebasing on another branch. I’ll fix it!